docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM alpine:latest
+
+MAINTAINER Sergio Correia <sergio@correia.cc>
+
+RUN addgroup -g 1000 opbot
```

Comment:
nit: 1000 is the starting uid for regular users on ubuntu variants. Maybe use something like 5000? This would make it easier to test locally, under certain situations.